rancher-config/docker-compose.yml

```diff
@@ -26,5 +26,5 @@ services:
         splunk-sourcetype: docker
         splunk-source: pharmacy-finder
```

Comment:
I know this hasn't been changed but do you think rather than `pharmacy-finder` the source should be `connecting-to-services`?